Tasks/Src/RTOS_BPS.c

```diff
@@ -10,6 +10,20 @@ void RTOS_BPS_SemPend(void) {
 void RTOS_BPS_SemPost(void) {
 
 }
+
+/**
+ * @brief Creates and returns a mutex.
+ * @param name - char* of the name of the mutex
+ * @return an BPS_OS_MUTEX object
+ */
+BPS_OS_MUTEX RTOS_BPS_MutexCreate(char* name) {
+    BPS_OS_MUTEX mut;
```

Comment:
This creates the mutex as a local variable (on stack), not sure if you can return mutexes by value

Reply:
You're probably right, will change